Source/hydro/riemann_nd.F90

```diff
@@ -1822,32 +1818,32 @@ subroutine HLLC(ql, qr, qpd_lo, qpd_hi, &
 
              if (S_r <= ZERO) then
                 ! R region
-                call cons_state(qr(i,j,k,:), U_state)
+                call cons_state(qr(i,j,k,:,comp), U_state)
```

Comment:
Same comment about the slicing here -- since in general this can be non-contiguous, I think we're better off avoiding it.

Reply:
oh you saw that ;)

Reply:
Since the slicing was already there, we could defer that change if you like.